net-p2p/createtorrent/createtorrent-1.1.4-r1.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools eutils
```

Comment:
Why is eutils needed?

Reply:
It is not needed. I had not noticed this. Thanks!

And I have removed it from the inherit line.